integrations/azure-devops/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,37 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: repository
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          blueprint: '"service"'
+          title: .name
+          properties:
+            url: '.url'
+            readme: file://README.md
+
+  - kind: repository-policy
+    selector:
+      query: '.type.displayName=="Minimum number of reviewers"'
```

Comment:
Regarding minimum approvers count, I've been able to change this to:
`.settings.minimumApproverCount != null`
Which seems more reasonable, But regarding work item linking it's more complicated, we'll need to decide what to do there / if we want to have it at all by default

Reply:
check if u can add more fields to the query

Reply:
So after a quick check there is a ID we can use.
Decided not to because I couldn't validate this ID is unique across all organizations and didn't find any data about it on the internet.
I believe that this is good enough, LMK if I need to reopen this comment 🙏